runtime/jvmti/jvmti_internal.h

```diff
@@ -1341,6 +1341,17 @@ suspendAgentBreakpoint(J9VMThread * currentThread, J9JVMTIAgentBreakpoint * agen
 UDATA
 findDecompileInfo(J9VMThread *currentThread, J9VMThread *targetThread, UDATA depth, J9StackWalkState *walkState);
 
+/**
+* A helper to walk a platform thread or virtual thread
+* @param[in] *currentThread current thread
+* @param[in] *targetThread the thread to walk
+* @param[in] threadObject the thread to walk
+* @param[in] walkState a stack walk state
+* @return a JVMTI error code
+*/
```

Comment:
Formatting: alignment. Also, see https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-function-parameters-clearly. For pointers, `*` is not required in the description.
```suggestion
/**
 * A helper to walk a platform thread or virtual thread
 * @param[in] currentThread current thread
 * @param[in] targetThread the thread to walk
 * @param[in] threadObject the thread to walk
 * @param[in] walkState a stack walk state
 * @return a JVMTI error code
 */
```